src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -219,6 +248,7 @@ class LibraryServiceGapicClient
         return self::$returnNameTemplate;
     }
 
+
```

Comment:
Nit: remove newline (if this is difficult because of snippet lang, then don't worry)